01_prepare_host.sh

```diff
@@ -112,10 +112,7 @@ elif [ "$(clusterctl version | grep -o -P '(?<=GitVersion:").*?(?=",)')" != "${C
 fi
 
 # Clean-up any old ironic containers
-for name in ironic ironic-inspector dnsmasq httpd mariadb ipa-downloader; do
-    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" kill $name
-    sudo "${CONTAINER_RUNTIME}" ps --all | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" rm $name -f
-done
+remove_ironic_containers
```

Comment:
If you look at the [install_requirements.sh ](https://github.com/metal3-io/metal3-dev-env/blob/746bf689dad5a83a9f9bbbdf6e18af677c004eab/ubuntu_install_requirements.sh#L7) you can see that it is sourcing lib/common.sh before it installs the docker/podman. I think that's the reason why the CI failed.

Reply:
Sorry ignore my comment, I didn't realize that you aren't actually calling the remove_ironic_containers func within the lib/common.sh